src/utils/convertSpecialCharacters/convertSpecialCharacters.test.ts

```diff
@@ -0,0 +1,9 @@
+import {convertSpecialCharacters} from './convertSpecialCharacters';
+
+describe('convertSpecialCharacters', () => {
+
+  it('should replace spacial characters', () => {
+    
+    expect(convertSpecialCharacters('ąęćńóśźżł')).toBe('aecnoszzl');
```

Comment:
**praise** nice!

Reply:
@kamil-borecki-cc  We added capital letters, is there anything else we missed?

Reply:
@darya-karpovich-cc sorry i misread this and i thought we are missing żź :D